site/en/docs/extensions/mv3/manifest/input_component/index.md

```diff
@@ -0,0 +1,9 @@
+---
```

Comment:
Poking through source, it looks like `input_component` is an array of objects. This section should be expanded to document the following properties.

- `name`: string. Required. Example: `"ToUpperIME"`
- `id`: string. Optional. Example: `"ToUpperIME"`
- `language`: string or array of strings. Optional. Examples: `"en"`, `["en", "pt"]`
- `layouts`: array of strings. Optional. Example: `["us::eng"]`. CrOS only supports one layout per input method. If multiple layouts are specified, selection order is undefined. Extensions are therefore strongly encouraged to  only specify one layout per input method. ([comment](https://source.chromium.org/chromium/chromium/src/+/main:chrome/browser/extensions/api/input_ime/input_ime_api_chromeos.cc;l=911-918;drc=24e5d915169bb4a4a0cff916ba1424fd6b31abcb))
  - A `xkb:` prefix indicates that this is a keyboard layout extension, but this only applies to components(?) ([source](https://source.chromium.org/chromium/chromium/src/+/main:ui/base/ime/ash/extension_ime_util.cc;l=162;drc=24e5d915169bb4a4a0cff916ba1424fd6b31abcb))
- `input_view`: string. Optional. If provided, this string must resolve to an extension resource.
- `options_page`: string. Optional. If provided, this string must resolve to an extension resource. If not provided, will fall back to the extension's options page.

**Resources consulted**

* [test manifest](https://source.chromium.org/chromium/chromium/src/+/main:chrome/test/data/extensions/input_ime/manifest.json;drc=06a8786f693747faedb42659008c2196733e26e1)
* [input_components_handler.cc](https://source.chromium.org/chromium/chromium/src/+/main:chrome/common/extensions/api/input_ime/input_components_handler.cc;drc=354945de1fb564ef04c07cf8bfedf434d2d81747)
* [input_ime_api_chromeos.cc](https://source.chromium.org/chromium/chromium/src/+/main:chrome/browser/extensions/api/input_ime/input_ime_api_chromeos.cc;l=931;drc=24e5d915169bb4a4a0cff916ba1424fd6b31abcb)

**Other notes**

- Third party IMEs cannot be used to log in to Chrome OS devices ([source](https://source.chromium.org/chromium/chromium/src/+/main:chrome/browser/extensions/api/input_ime/input_ime_api_chromeos.cc;l=931;drc=24e5d915169bb4a4a0cff916ba1424fd6b31abcb)).